auth-web/src/views/auth/PendingApprovalView.vue

```diff
@@ -1,26 +1,47 @@
 <template>
   <div>
-    <div v-if="$route.params.team_name">
-      <interim-landing :summary="$t('pendingInvitationTitle', { team: $route.params.team_name })" :description="$t('pendingInvitationMsg')" icon="mdi-information-outline">
+    <template v-if="isDenied">
+      <interim-landing :summary="$t('notAuthorized')" :description="$t('deniedInvitationMsg', { team: $route.params.team_name })" icon="mdi-alert-circle-outline" iconColor="error">
       </interim-landing>
-    </div>
-    <div v-if="!$route.params.team_name">
-      <interim-landing :summary="$t('noPendingInvitationTitle')" :description="$t('noPendingInvitationMsg')" icon="mdi-information-outline">
-      </interim-landing>
-    </div>
+    </template>
+    <template v-else>
+      <div v-if="$route.params.team_name">
+        <interim-landing :summary="$t('pendingInvitationTitle', { team: $route.params.team_name })" :description="$t('pendingInvitationMsg')" icon="mdi-information-outline">
+        </interim-landing>
+      </div>
+      <div v-if="!$route.params.team_name">
+        <interim-landing :summary="$t('noPendingInvitationTitle')" :description="$t('noPendingInvitationMsg')" icon="mdi-information-outline">
+        </interim-landing>
+      </div>
+    </template>
   </div>
 </template>
 
 <script lang="ts">
+import { Member, MembershipStatus } from '@/models/Organization'
 import { Component } from 'vue-property-decorator'
 import InterimLanding from '@/components/auth/InterimLanding.vue'
 import { KCUserProfile } from 'sbc-common-components/src/models/KCUserProfile'
+import OrgModule from '@/store/modules/org'
 import { Role } from '@/util/constants'
 import Vue from 'vue'
 import { mapState } from 'vuex'
 
 @Component({
-  components: { InterimLanding }
+  components: { InterimLanding },
+  computed: {
+    ...mapState('org', [
+      'currentMembership'
+    ])
+  }
 })
-export default class PendingApprovalView extends Vue { }
+
+export default class PendingApprovalView extends Vue {
+  protected readonly currentMembership!: Member
+  private isDenied: boolean = false
+
+  mounted () {
+    this.isDenied = !(this.currentMembership?.membershipStatus)
```

Comment:
Is `membershipStatus` undefined/null if the inviter has deactivated their account?

Reply:
Yes, it was a null response when its denied, thats why put the check like this.

Reply:
According to the TeamManagement mixin, this status should be `REJECTED`.  Part of this fix should include correcting the reason that is null at this stage.

Reply:
sorry I saw this now, I will check the backend for that and fix if anything not correct

Reply:
no problem! let me know if you need a hand

Reply:
@sutherlanda ..Was talking to @nitheesh-aot on this.. as of now , membership is returned only for active and pending members..  
Since this change makes sense , I think we can go ahead and return membership status for rejected users as well.. some more changes in API ,than expected it.. @nitheesh-aot is on it.. Thanks for the good review